.github/workflows/licenseCheck.yml

```diff
@@ -0,0 +1,52 @@
+name: Check licenses
```

Comment:
@nvdaes - I'm a bit concerned about this structure. I didn't expect separate workflows for each of these tasks. I'm particularly worried that releases will occur if other tests fail. The release should not be published if other tests fail. I think the previous structure of a single workflow is much better using jobs as mentioned here: https://github.com/nvaccess/nvda/pull/17728#issuecomment-2700074275

Reply:
Thanks for your review. I've unified the parallel jobs in an unique workflow. I'll add a release job which needs previous parallel jobs, and then I'll re-request another review.

Reply:
Thanks @nvdaes - this structure looks good